win32ss/gdi/gdi32/objects/bitmap.c

```diff
@@ -839,32 +810,6 @@ StretchDIBits(
 
     cjBmpScanSize = GdiGetBitmapBitsSize((BITMAPINFO *) pConvertedInfo);
 
-    if (lpBits)
```

Comment:
Similar comment to above: This might still need the reallocation if `lpBits` is not DWORD-aligned.